src/client/refresh/soft/sw_polyset.c

```diff
@@ -621,7 +645,7 @@ R_PolysetDrawSpans8_66(const entity_t *currententity, spanpackage_t *pspanpackag
 		}
 
 		pspanpackage++;
-	} while ((pspanpackage < triangles_max) && (pspanpackage->count != INT_MIN));
+	} while (pspanpackage->count != INT_MIN);
```

Comment:
I am expect that it will improve performance, but it looks as good as you have added additional `INT_MIN`, as for me it's good idea.

Reply:
I think the first part never evaluated to true. Anyway it was badly placed because we must check for overflow when adding spans, not when drawing them.